ocs_ci/ocs/ui/base_ui.py

```diff
@@ -85,22 +85,35 @@ def do_click(self, locator, timeout=30, enable_screenshot=False, copy_dom=False)
         enable_screenshot (bool): take screenshot
         copy_dom (bool): copy page source of the webpage
         """
-        try:
+        if (
+            version.get_semantic_version(get_ocp_version(), True)
```

Comment:
Where are we setting this default value to False? 
Also, wouldn't it be good to check ACM version and not OCP because we need these changes due to locators on ACM console?

Reply:
I remember that we had discussion on this, and you wanted a parameter based control on this but looks like the True OR False param may fail if we need to handle this per version, hence I left it as if/else. Regarding ACM version, we can't use ACM version as this is generic code and scenarios may not have ACM at all. Let me know what you think?

Reply:
If this change is stable for all existing UI tests too, I am okay with it.